kdesk/icon-hooks.sh

```diff
@@ -102,30 +102,29 @@ case $icon_name in
 
     "world")
         IFS=$'\n'
-        kano_statuses=`kano-profile-cli get_notifications_count`
-        is_online=`kano-profile-cli is_registered`
+        # Returns if online, prints notification count and stats activity
+        info=`kano-profile-cli get_world_info`
         apirc=$?
 
         if [ "$debug" == "true" ]; then
-            printf "Kano Profile API returns rc=$apirc, data=\n$kano_statuses\n"
+            printf "Kano Profile API returns rc=$apirc, data=\n$info\n"
```

Comment:
I get your point @convolu , but because the script is run for every single icon that's supposed to change behaviour, logging through kano-logs would be a massive slowdown. The intention of those prints is to actually debug the hooks during development from the console. kdesk will never call the hooks with debugging on